src/utils/create-utils.ts

```diff
@@ -333,11 +334,33 @@ export const getSecretObject = (values: SecretFormValues, namespace: string): Se
       const SSH_KEY = 'ssh-privatekey';
       data[SSH_KEY] = values.source[SSH_KEY];
     }
+  } else if (values.type === SecretTypeDropdownLabel.image) {
+    if (values.image.authType === ImagePullSecretType.ImageRegistryCreds) {
+      const dockerconfigjson = values.image.registryCreds.reduce(
+        (acc, cred) => {
+          acc.auths[cred.registry] = {
+            ...pick(cred, ['username', 'password', 'email']),
+            auth: Base64.encode(`${cred.username}:${cred.password}`),
+          };
+          return acc;
+        },
+        { auths: {} },
+      );
+
+      data = dockerconfigjson
+        ? { ['.dockerconfigjson']: Base64.btoa(JSON.stringify(dockerconfigjson)) }
```

Comment:
Why not use Base64.encode there? 
If encode also works well here, let us enjoy encode to keep  alignment with other usages.